demisto_sdk/commands/secrets/tests/secrets_test.py

```diff
@@ -174,6 +175,17 @@ def test_remove_white_list_regex(self):
         file_contents = self.validator.remove_whitelisted_items_from_file(white_list, file_contents)
         assert white_list not in file_contents
 
+    def test_remove_whitelisted_items_from_file_substring(self):
```

Comment:
this is not an integration test. please add test that will test the whole use case.
3 tests.
```
Use Case: secrets whitelisted via global secrets_white_list.json file
Given
- pack foo with foo.py in it. 
- foo.py file contains ip 4.4.4.4 in it.
- secrets_white_list.json contains a whitelisted ip 4.4.4.4

When 
- running the secrets function on the foo pack

Then
- ensure secret 4.4.4.4 is not matched as a secret
```
```
Use Case: secrets whitelisted via local .secrets_ignore file
Given
- pack foo with foo.py in it. 
- foo.py file contains ip 4.4.4.4 in it.
- foo pack contains .secrets_ignore that contains 4.4.4.4 whitelisted ip
- secrets_white_list.json 

When 
- running the secrets function on a foo pack

Then
- ensure secret 4.4.4.4 is not matched as a secret
```
```
Use Case: secrets is not whitelisted
Given
- pack foo with foo.py in it. 
- foo.py file contains ip 4.4.4.4 in it.
- foo pack contains .secrets_ignore that contains 2.2.2.2 whitelisted ip
- secrets_white_list.json contains a whitelisted ip 3.3.3.3

When 
- running the secrets function on foo pack

Then
- ensure secret 4.4.4.4 is matched as a secret
```

Reply:
the two last tests are already tested in the secrets_integration_tests.py